apps/vaporgui/RenderersPanel.cpp

```diff
@@ -18,10 +18,12 @@ RenderersPanel::RenderersPanel(VAPoR::ControlExec *ce)
     _splitter->setChildrenCollapsible(false);
     _inspectorRouter->Add(_renInspector = new RendererInspector(_ce));
     _inspectorRouter->Add(_dataInspector = new DatasetInspector(_ce));
+    setEnabled(false);
 }
 
 void RenderersPanel::Update()
 {
+    setEnabled(true);
```

Comment:
Again, this was already being handled and will this leads to crash documented in main comment.